src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -403,6 +403,40 @@ export default class TutorialView {
         });
 
         this.tutorial_step({
+            title: 'Say Route',
+            text: ['With the capability to edit the route, you obviously will need a way to know what their current route is. Typically, this is displayed',
+                   'in the flight progress strip. However, to preserve screen space, you can have the aircraft tell you their route with the say route command &lsquo;sr&rsquo;',
+                   'and the route will print out above the command bar.'
+               ].join(' '),
+            parse: (t) => {
+                if (prop.aircraft.list.length <= 0) {
+                    return t;
+                }
+
+                return t.replace('{CALLSIGN}', prop.aircraft.list[0].callsign);
+            },
+            side: 'left',
+            position: tutorial_position
+        });
+
+        this.tutorial_step({
+            title: 'Proceed Direct',
+            text: ['The proceed direct command &lsquo;pd&rsquo; instructs an aircraft to go directly to a waypoint in the flight plan.. For example, if an',
+                   'aircraft is flying to fixes [A, B, C], issuing the command "pd B" will cause the aircraft to go to B, then C. After flying ',
+                   'past the last fix, the aircraft will continue the flight plan as filed.'
```

Comment:
Let's get rid of the double dots at the end of the first sentence, and remove the last sentence (because it's explained in the previous slide).